modules/reference/pages/rpk/rpk-transform/rpk-transform-deploy.adoc

```diff
@@ -0,0 +1,51 @@
+= rpk transform deploy
+:description: rpk transform deploy
+
+Deploy a transform.

+

+When run in the same directory as a transform.yaml it will read the configuration file,

+then look for a .wasm file with the same name as your project. If the input and output

+topics are specified in the configuration file, those will be used, otherwise the topics

+can be specified on the command line using the --input-topic and --output-topic flags.

+

+Wasm files can also be deployed directly without a transform.yaml file by specifying it

+like so:

+

+  rpk transform deploy transform.wasm --name myTransform


```

Comment:
@rockwotj but dont you also need a number of other things like an input/output topic to do this successfully without the yaml?

We should show a real example here.

@micheleRP

Reply:
```
rpk transform deploy transform.wasm --name myTransform --input-topic my-topic-1 --output-topic my-topic-2
```

Reply:
`rpk transform` edits are in https://github.com/redpanda-data/docs/pull/104